core/src/main/java/hudson/security/AuthorizationStrategy.java

```diff
@@ -156,12 +192,32 @@ public boolean hasPermission(Authentication a, Permission permission) {
      * <p>
      * The default implementation returns {@link #getRootACL()}.
      *
+     * <p>
+     * Plugins may define this method
+     * but should not call this method directly,
+     * and should call {@link Cloud#getACL()} instead.
+     *
      * @since 1.252
      */
     public @Nonnull ACL getACL(@Nonnull Cloud cloud) {
         return getRootACL();
     }
 
+    /**
+     * Implementation can choose to provide different ACL for different {@link Node}s.
+     * This can be used as a basis for more fine-grained access control.
+     *
+     * <p>
+     * The default implementation returns {@link #getRootACL()}.
+     *
+     * <p>
+     * Plugins may define this method
+     * but should not call this method directly,
+     * and should call {@link Node#getACL()} instead.
+     *
+     * @param node
```

Comment:
Oops.
Added a commit to complete `param` and `return` of javadocs for `getACL`s.